crates/ruff/src/rules/pycodestyle/rules/bare_except.rs

```diff
@@ -7,6 +7,32 @@ use crate::source_code::Locator;
 use crate::violation::Violation;
 
 define_violation!(
+    ///## What it does
+    ///Checks for bare `except:` statements in `try...except`.
```

Comment:
nit: I don't think `except` is a statement.

Reply:
Good call. Tried to clarify.